docs/src/app/components/pages/components/date-picker.jsx

```diff
@@ -30,6 +30,12 @@ export default class DatePickerPage extends React.Component {
         name: 'Props',
         infoArray: [
           {
+            name: 'container',
+            type: 'one of: dialog, container',
+            header: 'default: dialog',
+            desc: 'The date pickers container type',
```

Comment:
@mbrookes Thanks for the feedback.

I would be happy to make it clearer, should I do it in a new PR?